docs/library/ubluetooth.rst

```diff
@@ -181,6 +182,21 @@ Event Handling
             elif event == _IRQ_MTU_EXCHANGED:
                 # MTU exchange complete (either initiated by us or the remote device).
                 conn_handle, mtu = data
+            elif event == _IRQ_L2CAP_COC_ACCEPT:
+                # A new channel has been accepted.
+                chan_handle, = data
```

Comment:
Are ACCEPT events needed at all?  Is there anything lost by just having `_IRQ_L2CAP_COC_CONNECT`?

Reply:
The idea is that the device can reject the connection. I didn't implement that in the first pass, but I have added that now. Return `0` or `None` to accept, non-zero integer to reject.

Possible values are NimBLE's `BLE_HS_E...` (see ble_l2cap_sig_ble_hs_err2coc_err). I think we should define our own mapping (I need to do this for IRQ_READ_REQUEST anyway, so will do that at the same time in a future PR).